web/client-api/src/test/java/io/deephaven/web/client/api/PartitionedTableTestGwt.java

```diff
@@ -116,15 +116,12 @@ public void testTickingPartitionedTable() {
 
                     return partitionedTable.getKeyTable().then(keyTable -> {
                         keyTable.setViewport(0, 99, keyTable.getColumns(), null);
-                        return keyTable.getViewportData().then(data -> {
-                            assertEquals(0d, keyTable.getSize());
-
-                            return waitForEventWhere(keyTable, JsTable.EVENT_UPDATED,
-                                    (CustomEvent<ViewportData> d) -> d.detail.getRows().length == 5, 20004);
-                        });
+                        return keyTable.getViewportData()
```

Comment:
still have a getViewportData() which ignores the data - this is effectively just a listener for the very next EVENT_UPDATED to go off.

Reply:
Good catch.